suricata.yaml.in

```diff
@@ -167,6 +167,19 @@ outputs:
             # payload-length: yes      # enable dumping payload length, including the gaps
             # packet: yes              # enable dumping of packet (without stream segments)
             # metadata: no             # enable inclusion of app layer metadata with alert. Default yes
```

Comment:
The line above looks a bit odd now

Reply:
From what I understood, Jeff kept both because you can enable or disable it, or you can enable and configure it (below). Not sure what the best way to represent that would be...